src/components/Header.tsx

```diff
@@ -1,7 +1,9 @@
 export default function Header() {
   return (
-    <div className="flex h-24 flex-row items-center justify-center">
-      <h1>Header</h1>
+    <div className="flex h-24 flex-row items-center justify-evenly">
+      <a href="http://localhost:3000">Home</a>
```

Comment:
these 3 hrefs are wrong it should be
`<a href="/">`
`<a href="/spreadsheet">`
`<a href="/planner">`